src/log-receiver/log-receiver.service.ts

```diff
@@ -87,4 +97,21 @@ export class LogReceiverService {
   async getLogsByServiceId(id : any) {
     return this.logModel.find({"serviceId": id}).exec();
   }
+  /**
+   * Connecting to kafka instance and waiting for incoming logs
+   * when a log comes in it is saved to the database
+   */
```

Comment:
this does not fell like a good comment.
something like 'connect to kafka instance and start consuming. incoming messages are saved to database' would be a fine summary sentence. 
and the detailed description could contain more info about which channel you connect to, and what kind of messages you read and to which database you safe them.

Reply:
ja, das wäre sehr gut, wenn du kennzeichnest das "logs" der name der collection ist :D

Reply:
diese kleinigkeit hier noch, dann gibt's von mir 'n approval :)

Reply:
Habs geändert :) benutze hier in diesem Service auch das .env file für die Kafka Url